Source/JavaScriptCore/inspector/agents/InspectorRuntimeAgent.cpp

```diff
@@ -203,15 +200,15 @@ Protocol::ErrorStringOr<std::tuple<Ref<Protocol::Runtime::RemoteObject>, std::op
         muteConsole();
     }
 
-    injectedScript.callFunctionOn(errorString, objectId, functionDeclaration, arguments ? arguments->toJSONString() : nullString(), returnByValue.value_or(false), generatePreview.value_or(false), result, wasThrown);
+    injectedScript.callFunctionOn(objectId, functionDeclaration, arguments ? arguments->toJSONString() : nullString(), returnByValue.value_or(false), generatePreview.value_or(false), awaitPromise.value_or(false), [callback = WTFMove(callback)] (Protocol::ErrorString& errorString, RefPtr<Protocol::Runtime::RemoteObject>&& result, std::optional<bool>&& wasThrown, std::optional<int>&&) {
```

Comment:
NIT: can you make the last parameter `std::optional<int>&& savedResultIndex` and then add a `UNUSED_PARAM(savedResultIndex)` so that it's more clear right here what it's for instead of having to look elsewhere to find out?

Reply:
Done.